### PR TITLE
Remove dead code as reported by linter

### PR DESCRIPTION
**Pull request by mem**
Signed-off-by: Marcelo E. Magallon <marcelo@sylabs.io>

### CODE REVIEW

**Review by cclerget — APPROVED** [review]
LGTM

File: pkg/util/capabilities/config_test.go [hunk 1]
```diff
@@ -75,25 +70,6 @@ func TestReadFromWriteTo(t *testing.T) {
 
 }
 
-func checkReadFrom(t *testing.T, confExpected Config, shouldFail bool) {
```
Comment (bauerm97) [inline pkg/util/capabilities/config_test.go:78]: Why are we deleting this here?

File: internal/pkg/remote/remote_test.go [hunk 2]
```diff
@@ -14,16 +14,6 @@ import (
 	useragent "github.com/sylabs/singularity/pkg/util/user-agent"
 )
 
-var remoteConfig = Config{
```
Comment (bauerm97) [inline internal/pkg/remote/remote_test.go:17]: Why is this dead code?

**Review by bauerm97 — CHANGES_REQUESTED** [review]

File: pkg/util/capabilities/config_test.go (hunk 1 above)
Comment (mem) [reply pkg/util/capabilities/config_test.go:78]: Nothing is using the function. :man_shrugging:

File: internal/pkg/remote/remote_test.go (hunk 2 above)
Comment (mem) [reply internal/pkg/remote/remote_test.go:17]: nothing is using the variable

**Review by bauerm97 — APPROVED** [review]

**Pull request merged**